pkg/build/api/types.go

```diff
@@ -411,6 +411,8 @@ const (
 	// BuildConfigLabelDeprecated was used as BuildConfigLabel before adding namespaces.
 	// We keep it for backward compatibility.
 	BuildConfigLabelDeprecated = "buildconfig"
+	// BuildConfigDeletionAnnotation is an annotation that marks a BuildConfig that is being deleted
+	BuildConfigDeletionAnnotation = "openshift.io/build-config.deletion-pending"
```

Comment:
> You'd have to delete the configChange trigger and set any image triggers to "automatic=false".

Right, I thought @deads2k was talking about DCs.

Upstream PR for inert deployments is https://github.com/kubernetes/kubernetes/pull/18901